openpype/hosts/houdini/api/lib.py

```diff
@@ -804,6 +805,45 @@ def get_camera_from_container(container):
     return cameras[0]
 
 
+def get_current_context_template_data_with_asset_data():
+    """
+    TODOs:
+        Support both 'assetData' and 'folderData' in future.
```

Comment:
Is `assetData` that's referred here is different from the `template_data["assetData"] = asset_data`  below ?

Reply:
"asset" is OpenPype naming, in AYON is replaced by "folder".